qemu.go

```diff
@@ -1009,6 +1033,9 @@ func (q *qemu) addDevice(devInfo interface{}, devType deviceType) error {
 	case blockDev:
 		drive := devInfo.(Drive)
 		q.qemuConfig.Devices = q.appendBlockDevice(q.qemuConfig.Devices, drive)
+	case vhostuserDev:
```

Comment:
Again this will be based on the actual device type

Reply:
@sboeuf why are we not doing a straight type switch here?

``
switch v := devInfo .(type) {
	default:
		fmt.Printf("I don't know about type %T!\n", v)
``